### DISCUSSION TITLE
veri havuzu oluşturulamadı hatası

### DISCUSSION THREAD

**Discussion by neslihanersari** [19/classroom#39] [Anonim] [answered]
![IMG_20220318_160431.jpg](https://user-images.githubusercontent.com/72813370/159011612-ceb7d959-de1e-4e3e-8013-96b12d8a44b9.jpg) .   Kurulumu hatasız bir şekilde tamamladım ödevi açarken onaylamalari da yaptım ama bu hatayı alıyorum. Ödev visual studio code da açılmıyor . Hatanın sebebini çözmeye çalıştım ama bulamadım. Ne yapmam gerekiyor?

**Comment by roktas** [comment, marked as answer]
Ödev depon burada: https://github.com/19-22s-ruby-2/point-distance-neslihanersari

(Ödevlerinizi bu kadar geç zamana neden bırakıyorsunuz?)

**Reply by neslihanersari** [reply]
Teşekkür ederim hocam link üzerinden tekrar deneyince oldu .
